platform/install/configure/config.mdx

```diff
@@ -93,6 +93,30 @@ config:
       X-XSS-Protection: 1; mode=block
 ```
 
+### Setting additional certificate authority for Platform
+If you decide to configure vCluster Platform helm release with  `additionalCA`, it will be automatically used by all vClusters managed by the vCluster Platform to establish a connection between vClusters and the Platform. You don't need to tweak individual vCluster configuration to enable this.
```

Comment:
Update to: 
```
If you want to have **all** your virtual clusters managed by the vCluster Platform to use a specific certificate authority to establish the connection between the virtual cluster and the platform, you can set it as a configuration option on the vCluster Platform instead of configuring it on each indiviudal virtual cluster.
```

Note about the docs: You can't pluralize vCluster so it must be virtual clusters.